pkgs/applications/networking/instant-messengers/rambox/default.nix

```diff
@@ -12,6 +12,17 @@ let
     };
     sencha = callPackage ./sencha {};
   };
+  # https://github.com/NixOS/nixpkgs/pull/32741#issuecomment-352203170
+  electron_1_7 = electron.overrideAttrs (oldAttrs: rec {
```

Comment:
maybe also add a check to reevaluate this decision in future versions of rambox.